pointcloud/octree/basic.go

```diff
@@ -0,0 +1,171 @@
+package octree
+
+import (
+	"context"
+
+	"github.com/edaniels/golog"
+	"github.com/golang/geo/r3"
+	"github.com/pkg/errors"
+
+	pc "go.viam.com/rdk/pointcloud"
+)
+
+// octree data is the structure of an octree.
+type basicOctree struct {
+	ctx    context.Context
```

Comment:
It was an input into the New Function as defined by the tech spec. I figured it would come more into play when we move to serialized octree but will remove it for now until there is a need.